src/frontend/xgboost/xgboost.h

```diff
@@ -7,10 +7,17 @@
 #ifndef TREELITE_FRONTEND_XGBOOST_XGBOOST_H_
 #define TREELITE_FRONTEND_XGBOOST_XGBOOST_H_
 
+#include <string>
+#include <vector>
 #include <cmath>
 
 namespace treelite {
+
+struct ModelParam;  // forward declaration
+
 namespace details {
+namespace xgboost {
```

Comment:
`ProbToMargin` seems like a struct that is generally useful (beyond just xgboost). Do we want it in the `xgboost` namespace?

Reply:
Only XGBoost requires ProbToMargin conversion on the global bias field.